{"testcasedatas":[{"firstjacocotime":5.006,"tagname":"preference_pre_reverse","firstexecutionsuccess":false,"firsttestcasepath":"\\testcase\\allpreferencecases\\precase_log.py","firstconsumedtime":191.291,"firstlocpath":"\\exeresult\\interestresult\\preference_pre_reverse.txt","firstcoveragepath":"\\coverage\\interestallcoverage\\coveragepreference_pre_reverse.ec","firsterrorlog":"FAILstr(e):\t\tMessage: An unknown server-side error occurred while processing the command. Original error: The swipe did not complete successfullyrepr(e):\tWebDriverException()Traceback (most recent call last):  File \"C:\\Users\\yifeiLu\\Documents\\TestingProjectHome\\antennapod\\testcase\\allpreferencecases\\precase_log.py\", line 278, in <module>    clickInList(driver, \"new UiSelector().text(\\\"Sort alphabetically\\\")\")  File \"C:\\Users\\yifeiLu\\Documents\\TestingProjectHome\\antennapod\\testcase\\allpreferencecases\\precase_log.py\", line 111, in clickInList    element = scrollToFindElement(driver, str)  File \"C:\\Users\\yifeiLu\\Documents\\TestingProjectHome\\antennapod\\testcase\\allpreferencecases\\precase_log.py\", line 79, in scrollToFindElement    swipe(driver, 0.5, 0.55, 0.5, 0.2)  File \"C:\\Users\\yifeiLu\\Documents\\TestingProjectHome\\antennapod\\testcase\\allpreferencecases\\precase_log.py\", line 65, in swipe    end_y=int(height * endyper), duration=1000)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 548, in swipe    action.perform()  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\common\\touch_action.py\", line 96, in perform    self._driver.execute(Command.TOUCH_ACTION, params)  File \"C:\\Python27\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute    self.error_handler.check_response(response)  File \"C:\\Python27\\lib\\site-packages\\appium\\webdriver\\errorhandler.py\", line 29, in check_response    raise wdeWebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: The swipe did not complete successfully"},{"firstjacocotime":5.003,"tagname":"preference_pre_default","firstexecutionsuccess":true,"firsttestcasepath":"\\testcase\\allpreferencecases\\defaultprecase_log.py","firstconsumedtime":212.59,"firstlocpath":"\\exeresult\\interestresult\\preference_pre_default.txt","firstcoveragepath":"\\coverage\\interestallcoverage\\coveragepreference_pre_default.ec","firsterrorlog":null}],"interestmap":[]}